ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
The mb_ereg(i) docs have been a mess wrt. $regs parameter and the return value. I've already changed this (will take a while to get distributed, though) to match the respective parts of the [ereg(i) docs](http://php.net/manual/en/function.ereg.php), which describe the current behavior rather well. Nonetheless, the type declaration of the $regs parameter in the function prototype is array, what doesn't match the current behavior. However, the function prototype doesn't match the function's behavior anyway, what can be observed with `strict_types=1`.